devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -150,7 +150,18 @@ def configure() {
     sendEvent(name: "checkInterval", value: 2 * 60 + 10 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
     log.debug "Configuring Reporting"
     return refresh() +
-    	   zigbee.onOffConfig() +
-           zigbee.configureReporting(zigbee.SIMPLE_METERING_CLUSTER, ATTRIBUTE_READING_INFO_SET, DataType.UINT48, 1, 600, 1) +
-           zigbee.electricMeasurementPowerConfig(1, 600, 1) 
+            zigbee.onOffConfig() +
+            zigbee.configureReporting(zigbee.SIMPLE_METERING_CLUSTER, ATTRIBUTE_READING_INFO_SET, DataType.UINT48, 1, 600, 1) +
+            zigbee.electricMeasurementPowerConfig(1, 600, 1) +
+            additionalConfig

```

Comment:
Won't this append `null` for non-Dawon devices? I don't know whether that will cause problems upstream or not.

Reply:
unless there's an example of us doing something similar elsewhere in the codebase

Reply:
Honestly, I'm not sure. I guess it'll be easier for me to just do it "sure way".